### PR TITLE
Clarify that you can use -d and --rm together now

### PR DESCRIPTION
**Pull request by mstanleyjones**
You can use `-d` and `--rm` together now.

Fixes https://github.com/docker/docker.github.io/issues/3666

baby fox biting mom's tail

### CODE REVIEW

**Comment by mstanleyjones** [comment]
I have no idea why it modified whitespace in those tables. Maybe tabs to spaces??

File: docs/reference/run.md [hunk 1]
```diff
@@ -149,7 +150,7 @@ is receiving its standard input from a pipe, as in:
 The operator can identify a container in three ways:
 
 | Identifier type       | Example value                                                      |
-| --------------------- | ------------------------------------------------------------------ |
+|:----------------------|:-------------------------------------------------------------------|
```
Comment (thaJeztah) [inline docs/reference/run.md]: Looks like it's the `left align` (`|:-` making the difference)

And aligning the right-hand border for the other tables

Comment (thaJeztah) [inline docs/reference/run.md]: Anyway; adding `?w=1` to the URL to hide whitespace changes makes it easier to view; https://github.com/docker/cli/pull/290/files?w=1

**Comment by mstanleyjones** [comment]
Should this get cherry-picked?

**Pull request merged**